src/components/Cell.tsx

```diff
@@ -414,13 +419,20 @@ const Cell: React.FC<CellInterface> = ({
         underlayColor={highlightUnderlayColor || theme.colors.body}
         onPressIn={onHighlightRow}
         onPressOut={onUnHighlightRow}
-        testID={testID}>
+        testID={testID}
+        accessibilityHint={accessibilityHint}
+        accessibilityRole="button">
```

Comment:
Is button the only possible role or would it be more sufficient to define a prop with default value `button`?